### ISSUE TITLE
Fails to download the model from Huggingface

### ISSUE THREAD

**Issue by flesler** [TheLastBen/fast-stable-diffusion#1436]
![image](https://user-images.githubusercontent.com/38258/213478757-0403cf11-bc87-40ab-b73b-acfa9d803f5a.png)

**Comment by flesler** [comment]
Fixed it on my end, with quick ugly patch:
```
origin = f"https://USER:{token}@huggingface.co/{Path_to_HuggingFace}"
!git remote add -f origin $origin
```

**Comment by flesler** [comment]
Yeah I noticed that and patched the code for myself. Having it in the notebook was way more practical IMO.
Even if that fixes, the way it fails right now is super cryptic. Rather than checking if there's no token when needed and showing a clear error message

**Comment by TheLastBen** [comment]
My mistake, fixed now https://github.com/TheLastBen/fast-stable-diffusion/commit/a13fea5ffb786795667ffa75b8c34ba96df0f0dd

**Comment by katyes75** [comment]
e}"
> !git remote add -f origin $origin
> ```



> 



> Lo arreglé en mi extremo, con un parche feo rápido:
> 
> ```
> origin = f"https://USER:{token}@huggingface.co/{Path_to_HuggingFace}"
> !git remote add -f origin $origin
> ```

**Comment by katyes75** [comment]
@flesler I'm new to this, where did you put the patch to fix the token?

**Comment by flesler** [comment]
You can create the file as described in there, or add a cell that has:
```
token = '...your token...'
```
and run it before downloading the model
